examples/test/PivotTableSizing_test.js

```diff
@@ -40,3 +40,32 @@ test("should trigger resize after attribute filter change", async t => {
     const newWidth = await Selector(tableWrapper).find(firstAttributeCellSelector).clientWidth;
     await t.expect(newWidth).lt(originalWidth);
 });
+
+async function resizeTable(t, tableQuerySelector, width) {
+    await t.eval(
+        () => {
+            document.querySelector(tableQuerySelector).style.width = width;
+        },
+        { dependencies: { tableQuerySelector, width } },
+    );
+}
+
+async function resizeAndVerifyColumnVisibility(t, tableSelector, columnSelector) {
+    const tableElement = Selector(tableSelector);
+    const columnElement = await tableElement.find(columnSelector);
+    await resizeTable(t, tableSelector, "600px");
```

Comment:
is this middle step and expect needed?

Reply:
Resize not necessarily, but I would keep the `expect` there. I want to be sure it tests a newly shown column, otherwise it could generate false-positive results.